elytron-oidc-client/subsystem/src/main/resources/org/wildfly/extension/elytron/oidc/LocalDescriptions.properties

```diff
@@ -1,144 +1,142 @@
-
-
+elytron-oidc-client.secure-deployment.verify-token-audience=If set to 'true', then during bearer-only authentication, verifies if the token contains the client name defined as an audience. It is recommended to set to 'true' for improved security.
+elytron-oidc-client.secure-deployment.adapter-state-cookie-path=If set, this defines the path used in cookies set by the subsystem. This is useful when deploying an application in the root context path.
+elytron-oidc-client.secure-deployment.credential=The credential used to communicate with the OpenID Connect provider.
+elytron-oidc-client.secure-deployment.credential.add=Add credentials used to communicate with the OpenID provider.
+elytron-oidc-client.secure-deployment.credential.remove=Remove credentials used to communicate with the OpenID provider.
+elytron-oidc-client.secure-deployment.credential.client-key-password=The password for the client key when the credential provider is used. This is required if client-keystore-file has been specified.
+elytron-oidc-client.secure-deployment.credential.client-keystore-password=The password for the client keystore when the credential provider is used. This is required if client-keystore-file has been specified.
+elytron-oidc-client.secure-deployment.credential.client-keystore-file=The path to the client keystore when the credential provider is used. This is optional.
+elytron-oidc-client.secure-deployment.credential.client-keystore-type=The client keystore type when the credential provider is used.
+elytron-oidc-client.secure-deployment.credential.client-key-alias=The client key alias when the credential provider is used.
+elytron-oidc-client.secure-deployment.credential.token-timeout=The amount of time after which the token expires and can no longer be used to authenticate requests.
+elytron-oidc-client.secure-deployment.credential.secret=The client secret that was registered with the OpenID provider.
+elytron-oidc-client.secure-deployment.credential.algorithm=Specify the credential signature algorithm used by the OpenID provider.
+elytron-oidc-client.secure-deployment.redirect-rewrite-rule.add=Add the rewrite rule for the redirect URI. The rewrite rule is an object notation, where the key is a regular expression with which the redirect URI is matched and the value is the replacement String. You can use the '$' character to use for back-reference in the replacement String.
+elytron-oidc-client.secure-deployment.redirect-rewrite-rule.remove=Remove the rewrite rule for the redirect URI.
+elytron-oidc-client.secure-deployment.redirect-rewrite-rule.replacement=The replacement String in the rewrite rule for the redirect URI.
+elytron-oidc-client.secure-deployment.socket-timeout-millis=The timeout for the socket waiting for data after establishing the connection, in milliseconds. A timeout value of zero is interpreted as an infinite timeout, negative value is interpreted as undefined.
```

Comment:
s / , in milliseconds / in milliseconds
s / , negative value / , and a negative